src/runtime/process.rs

```diff
@@ -623,7 +623,7 @@ where
     Ok(value)
 }
 
-static EXEC_CPU_AFFINITY_REGEX: Lazy<Regex> = Lazy::new(|| {
+static EXEC_CPU_AFFINITY_REGEX: LazyLock<Regex> = LazyLock::new(|| {
```

Comment:
This can be made to work on older versions of rust, while still dropping the external `once_cell` dep by having a wrapper function to access the static.

There's an example of this pattern here e.g.: https://github.com/containers/containers-image-proxy-rs/commit/10e0fea73cc4c4078f50081a813768895ce707e0